packages/teleterm/src/ui/TabHost/useNewTabOpener.ts

```diff
@@ -1,25 +1,20 @@
-import { useAppContext } from 'teleterm/ui/appContextProvider';
 import { useCallback } from 'react';
+import { DocumentsService } from 'teleterm/ui/services/workspacesService';
 
-export function useNewTabOpener() {
-  const ctx = useAppContext();
-
-  const documentsService =
-    ctx.workspacesService.getActiveWorkspaceDocumentService();
-
-  const localClusterUri =
-    ctx.workspacesService.getActiveWorkspace()?.localClusterUri;
-
+export function useNewTabOpener(params: {
+  documentsService: DocumentsService;
+  localClusterUri: string;
+}) {
```

Comment:
I'd be for not using the name `params` here at all and destructuring those args instead, but that can be fixed in the future.

Reply:
Done now :)